oaknational/Oak-Web-Application
# fix: 404 for canonical lesson page [LESQ-673]

## Description

Music year: 1984

- add error handling to canonical lesson page
- add error reporting to legacy canonical lesson mv


## Issue(s)

Fixes #
500 page for lessons not found

## How to test
404 pages are not rendering in the preview env so not sure how to test...

1. Go to https://deploy-preview-2287--oak-web-application.netlify.thenational.academy/teachers/lessons/nonsense-lesson
3. You should see a 404 page rather than 500
